docs/OV_Runtime_UG/model_representation.md

```diff
@@ -58,18 +58,18 @@ OpenVINO™ Runtime provides two types for shape representation:
 
 @endsphinxtabset
 
-  But in most cases before getting static shape using `get_shape()` method, you need to check that shape is static.
+However, in most cases before getting static shape using `get_shape()` method, you need to check that shape is static.
 
 ## Operations
 
 The `ov::Op` class represents any abstract operation in the model representation. Use this class to create [custom operations](../Extensibility_UG/add_openvino_ops.md).
 
 ## Operation Sets
 
-Operation set (opset) is a collection of operations that can be used to construct a model. The `ov::OpSet` class  provides a functionality to work with operation sets.
+Operation set (opset) is a collection of operations that can be used to construct a model. The `ov::OpSet` class provides a functionality to work with operation sets.
 For each operation set, OpenVINO™ Runtime provides a separate namespace, for example `opset8`.
 Each OpenVINO™ Release introduces new operations and add these operations to a new operation set. New operation sets help to introduce a new version of operations that change behavior of previous operations. Using operation sets allows you to avoid changes in your application if new operations have been introduced.
```

Comment:
```suggestion
Each OpenVINO™ Release introduces new operations and adds them to a new operation set. These new sets help to introduce a new version of operations that change behavior of previous operations. Using operation sets allows you to avoid changes in your application if new operations have been introduced.
```